src/lib.rs

```diff
@@ -0,0 +1,37 @@
+use ursa::{
+    keys::{KeyGenOption, PrivateKey},
+    signatures::{ed25519::Ed25519Sha512, SignatureScheme},
+};
+
+/// This library contains utilities to work with cryptographical keys.
+
+/// If you need a keypair, use this method by default.
+/// It uses [Ursa](https://docs.rs/ursa) library with `Ed25519` solution of [EdDSA](https://tools.ietf.org/html/rfc8032).
+/// [OsRng](https://docs.rs/rand/0.7/rand/rngs/struct.OsRng.html) random number generator is used.
+pub fn generate_keypair() -> Result<(Vec<u8>, Vec<u8>), String> {
```

Comment:
If you know the length and it is immutable, why use Vec?

Reply:
By analogy with `ursa` API - will check if such a guarantees provided by their functions.

Reply:
Moved to an issue #2